mkdocs/docs/HPC/teaching_training.md

```diff
@@ -0,0 +1,108 @@
+# Teaching and training
+
+The HPC infrastructure can be used for teaching and training pruposes, and
+we provide support with getting you organised.
+
+We assume that the course requirements are such that the interactive cluster
+can be used. If these are insufficient, you will need to request a reservation.
+
+[^TODO]: insert some links to the interactvie cluster?
+
+In order to help you out, you need to provide following pieces of information
+when you [contact the {{ hpcteam }}]({{ hpc_support_url }}):
+
+* Group moderators and optional UGent course id
+* Start and end date
+* Optional
+  * Share
+  * Reservation
+  * Software
+  * Portal app
+
+## Group moderators
+
+To organise teachers and students, groups are used. Typically the teachers map
+to the role of moderator(s), and students are regular members.
+
+The moderator can invite
+students as member via their vsc account (the students have to accept the invite);
+or the students can invite themself to the group (and moderator has to approve the invites).
+
+You provide us with the vsc account(s) for the moderator(s),
+the moderators can manage the members and moderators themself afterwards.
+
+[^TODO]: do we support initial list of participants?
+
+[^TODO]: we control the groupname prefix? or have them all use the same one (unprotected). incl some convntion like academci year
+
+
+### UGent course id
+
+[^TODO]: check terminology
+
+When teaching a registered UGent course, we can offload the management of the vsc accounts from the students and
+the teachers. We do this by creating a vsc account for each registered student that is linked to the course id
+(if they do not have an account already). And we then populate the group with the vsc accounts of the students
+and the list of moderators. Using this method, you cannot manage the members or moderators yourself:
+group members unknown to the course will be considered unregistered students and will be removed from the group.
+
+The created accounts will be accounts without an ssh-key. This allows the students to use e.g. the portal, but if
```

Comment:
`VSC accounts`

`ssh-key` -> `SSH public key`

`portal` -> `web portal` (and link to the corresponding page)